crmsh/sbd.py

```diff
@@ -10,6 +10,7 @@
 from . import xmlutil
 from . import watchdog
 from . import parallax
+from . import healthcheck
```

Comment:
It does not make sense to add a dependency to module `healtcheck` just for exception `FixFailure`. It is just a ordinary subclass of `Exception` without any extra features.